microsoft-365/managed-desktop/service-description/privacy-personal-data.md

```diff
@@ -14,133 +14,69 @@ audience: Admin, ITPro
 
-Microsoft Managed Desktop stores its data in one or more of the following Microsoft storage services:
+Microsoft Managed Desktop Engineering Operations and Security Operations teams are located in the United States and India. 
 
-- Azure SQL
-- Azure storage
-- Dynamics 365
+## Microsoft Windows 10 diagnostic data
 
-Microsoft Managed Desktop stores its data in the United States. Personal data is retained by Microsoft Managed Desktop for a maximum of 30 days, except for alert data for Microsoft Managed Desktop devices collected by Microsoft Defender for Endpoint. The actual alert data (which could include personal data) is stored for 180 days. Alert data with personal data removed is stored for up to two years. In compliance with the General Data Protection Regulation (GDPR) and California Consumer Privacy Act (CCPA), Microsoft Managed Desktop honors the data subject rights for any personal data that is stored in alert data.
+Microsoft Managed Desktop uses [Windows 10 Enhanced diagnostic data*] (https://docs.microsoft.com/windows/privacy/windows-diagnostic-data) to keep Windows secure, up to date, troubleshoot problems, and make product improvements. The enhanced diagnostic data setting includes more detailed information about the devices enrolled in Microsoft Managed Desktop and their settings, capabilities, and device health. When enhanced diagnostic data is selected, data including required diagnostic data are collected. See Changes to Windows diagnostic data collection - Windows Privacy | Microsoft Docs for more information about the Windows 10 diagnostic data setting and data collection.
 
-### Staff location
+*Note that in the future version of Windows, the telemetry taxonomy will change.  Microsoft Managed Desktop is committed to processing only the data that the service needs.  While this will mean the diagnostic level will change to Optional, Microsoft Managed Desktop will implement the limited diagnostic policies to fine tune diagnostic data collection required for the service.  More details, see [Changes to Windows diagnostic data collection - Windows Privacy | Microsoft Docs] (https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fwindows%2Fprivacy%2Fchanges-to-windows-diagnostic-data-collection&data=04%7C01%7CSaki.Tong%40microsoft.com%7C5e8cc0a1356b43b18a3308d92c4b375e%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637589523823867011%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=LD4FD%2BWEQ13%2BU1eCBLtKZiy8HZLgqCnZWldaNhs%2FeaM%3D&reserved=0) 
```

Comment:
```suggestion
*Note that in future versions of Windows, the telemetry taxonomy will change.  Microsoft Managed Desktop is committed to processing only the data that the service needs.  While this will mean the diagnostic level will change to Optional, Microsoft Managed Desktop will implement the limited diagnostic policies to fine-tune diagnostic data collection required for the service.  For more details, see [Changes to Windows diagnostic data collection] (/windows/privacy/changes-to-windows-diagnostic-data-collection).
```